src/components/navbar/Navbar.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './Navbar.css';
+
+function Navbar () {
+    return (
+
+            <div className ='navi'>
```

Comment:
- You don't need a div around the `<nav>` element
- Use the component's name as a className on the wrapping element, e.g. `NavBar` instead of `navi`